source/katana/install.md

```diff
@@ -0,0 +1,48 @@
+---
+title: sabre/katana
+product: katana
+layout: default
+---
+
+If you downloaded sabre/katana as an archive, skip the pre-requisites.
+
+Pre-requisites
+--------------
+
+To grab dependencies of the project, make sure you have [Composer] installed,
+and then run:
+
+    composer install
+
+Also, make sure you have [Bower] installed, and then run:
+
+    bower install
+
+Then, to install sabre/katana, you have two options.
+
+In your browser
+---------------
+
+You need to start an HTTP server; example with the PHP built-in server:
+
+    php -S 127.0.0.1:8888 -t public public/.webserver.php
```

Comment:
Using the built-in PHP webserver must not be the default. It doesn't make sense for most of the population. One of the key things we wanted to do with katana was to make it easier to use. This means no command line at all.